mylist.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(*args)
+    @list = args
+  end
```

Comment:
- [ ] Optional suggestions

- `#initialize` method accepts multiple arguments. so I would recommend doing this

````
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```
`

Reply:
`*args` will accept all the arguments; from my understanding therefore `args` itself will be an array. I believe What they have already is more optimal.

Reply:
Thanks for the feedback